### DISCUSSION TITLE
404 error

### DISCUSSION THREAD

**Discussion by antbaguley79** [community/community#129405] [New to GitHub]
I can't work our why I am getting a 404 error for my page. I have uploaded all of my folders and files, there is an index.html file in the root directory, I have ensured the pages settings have been updated, have I missed something?

My page link is https://antbaguley79.github.io/Tonis-Touch/

**Comment by Nimittxo** [comment]
These Issues often occur when You use a build process for your HTML file, By checking the file path, clearing your browser cache, and making sure any building instructions are correct, you should be able to fix the 404 error and get your Toni's Touch webpage up and running!

**Comment by manuel14mds** [comment]
This is the link: https://antbaguley79.github.io/Tonis-Touch/tonis_touch_cleaning/

If you want to access with te link you share, move all the contents from the 'tonis_touch_cleaning' folder to the root directory.

@antbaguley79

**Comment by Tshiring** [comment]
Make a file with index.html and try again also check in pages whether you have selected root or any folder
